gunrock/util/type_traits.hxx

```diff
@@ -2,27 +2,27 @@
 
 #include <type_traits>
 
-namespace std {
+// namespace std {
```

Comment:
I committed a fix for this, please pull from master. We don't have to comment these, master is now on `C++17`.